extension/war/src/main/webapp/groovy/social/webui/UIUserInvitation.gtmpl

```diff
@@ -1,5 +1,6 @@
 <%
 String inviteAction = uicomponent.event("Invite") + "; return false;";
+String inviteSpaceMember = _ctx.appRes("UISpaceMember.button.invite");
```

Comment:
Do we already have the value of this key ?

Reply:
Absolutely ([link](https://github.com/Meeds-io/social/blob/develop/component/webui/src/main/resources/locale/social/Webui_en.properties#L757))